scene/gui/text_edit.cpp

```diff
@@ -4174,7 +4174,7 @@ Control::CursorShape TextEdit::get_cursor_shape(const Point2 &p_pos) const {
 		}
 	}
 
-	return CURSOR_IBEAM;
+	return Control::get_default_cursor_shape();
```

Comment:
Is the `Control::` scope useful here? We inherit it so it's directly available.

Reply:
You're right, I was thinking that get_default_cursor_shape was overwrited in this class but it wasn't.